kentikapi/credentials.go

```diff
@@ -0,0 +1,88 @@
+package kentikapi
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+type Profile struct {
+	AuthEmail string `json:"email"`
+	AuthToken string `json:"api-key"`
+}
+
+const DefaultProfile = "default"
+
+// GetCredentials reads and returns (email, token) pair from environment variables or specified profile config file.
+// Environment variables used: KTAPI_AUTH_EMAIL, KTAPI_AUTH_TOKEN.
+// Default profile name can be specified by global variable DefaultProfile.
+// Profile config file path is ${HOME}/.kentik/default by default. It can be overridden by
+// KTAPI_CFG_FILE env variable or KTAPI_HOME variable used to path "${KTAPI_HOME}/.kentik/<profile_name>".
```

Comment:
Would be nice to have a comment like this:

Credentials lookup order (assuming profile name is "default"):
1. env vars: KTAPI_AUTH_EMAIL, KTAPI_AUTH_TOKEN
2. file at $KTAPI_CFG_FILE
3. file at $KTAPI_HOME/.kentik/default
4. file at $HOME/.kentik/default
5. file at ./.kentik/default

Reply:
Done